lib/helpers/clone.js

```diff
@@ -112,10 +112,6 @@ function clone(obj, options, isArrayChild) {
 }
 module.exports = clone;
 
-/*!
```

Comment:
This is used to avoid automatically generating docs for `cloneObject` since it's meant to be used internally and not part of the public API.